crypto/rand/rand_lcl.h

```diff
@@ -20,8 +20,16 @@
 /* How many times to read the TSC as a randomness source. */
 # define TSC_READ_COUNT                 4
 
-/* Maximum count allowed in reseeding */
-# define MAX_RESEED                     (1 << 24)
+/* Maximum reseed intervals */
+# define MAX_RESEED_INTERVAL                     (1 << 24)
+# define MAX_RESEED_TIME_INTERVAL                (1 << 20) /* approx. 12 days */
```

Comment:
This is a somewhat arbitrary choice. On one hand, I did not want to admit ridiculously large reseed time intervals, on the other hand I did not want to be too strict.